modules/doc/content/framework_development/authorship_guidelines.md

```diff
@@ -0,0 +1,74 @@
+
+The contributors are listed in two categories.
+
+- Semi-permanent contributors are included in every new MOOSE framework paper by default. These contributors
+  have written a very large part of the MOOSE framework and must be credited, at least until their contributions
+  are overtaken by others.
+
+The current criteria for semi-permanent authorship are listed below:
+
+- Top 10 contributor as measured by lines of code in the `framework/` and other utility folders or
+- Top 10 contributor as measured by number of commits in the `framework/` folder
+
+
+!alert note
+While it seems poor practice to recognize contribution by lines of code or commit numbers, this criteria did not
+exist when these contributions were made, so the metric did not influence the coding style of these authors.
+These criteria may be reformulated in the future in terms of contributed features and impact.
+
+
+- Transient contributors are included in the next MOOSE framework paper for the period(s) in which they contributed.
+  These contributors have played an important role in MOOSE development since the previous paper was published.
+
+
+The current guidelines for transient authorship are listed below:
+
+- 1 major contribution or
+- 4 minor contributions and significant involvement in MOOSE-related activities or
+- 10 fixup contributions and significant involvement in MOOSE-related activities
+
+
+Contributions are only tallied for the *framework and supporting utilities*. Modules are deliberately excluded,
+as developers of a module should publish a module paper and report on new developments outside of the MOOSE framework paper.
+MOOSE-related activities are developments in the MOOSE modules and MOOSE applications.
+
+A list of major, minor and fixup example contributions has been developed as guidelines for
+classification and can be shared upon request. The classification of contributions is decided when
+gathering the list of authors for the next paper.
+
+!alert note
+Contributors who do not meet the criteria for authorship in the paper will see their contributions from
+the past period considered for the next paper.
+
+!alert note
+Code reviews and user support are currently not formalized in the criteria. Depending on the scale of the effort,
+we may consider them at the level of minor and fixup contributions.
+
+!alert note
+The contribution guidelines may be scaled depending on the time elapsed between successive publications. The current
+guidelines were developed for an 18 months time frame between two publications.
+
+## MOOSE Modules paper authorship guidelines
+
+Modules should strive to follow the same principles as the framework with regards to authorship inclusivity
+and representativity. However, most modules are not developed by the framework team and have their own policies.
+Please contact the relevant module team for more information.
```

Comment:
```suggestion
Please contact the relevant module code owners for more information.
```

Reply:
I think this makes it more tangible who one should contact

Reply:
I ll add a note. I dont consider code owners to be close enough to the team for each module

Reply:
see now

Reply:
> I dont consider code owners to be close enough to the team for each module

Lol, what?! Then we don't have the correct code owners

Reply:
I would hope that the code owners listed are good representatives of the module teams. If you're saying that not all module team members are listed as code owners, then I would agree with that.

Reply:
I like what you wrote, so I'm resolving this convo

Reply:
Reactor team at ANL has no code owner.

Fluid properties is similarly a not full match of people in charge. 

I m sure tensor mechanics and phase field similarly have 1/2 the people we actually consider to be decision makers for those efforts in the codeowners file

Reply:
Ah i started typing too early. Yes you got what I mean